.github/workflows/dependency-review.yaml

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+---
+name: 'Dependency Review'
```

Comment:
Lets keep this lower case for consistency with other workflows. Also quoting strings is not needed in yaml.

Reply:
done.